android/java/org/chromium/chrome/browser/util/BraveReferrer.java

```diff
@@ -36,6 +36,7 @@ public class BraveReferrer implements InstallReferrerStateListener {
     private static final String APP_CHROME_DIR = "app_chrome";
     private static final String PROMO_CODE_FILE_NAME = "promoCode";
     private static final String BRAVE_REFERRER_RECEIVED = "brave_referrer_received";
+    private static final String GOOGLE_ADS_REFERRAL_CODE = "UAC001";
```

Comment:
how did we end up with that particular code?

Reply:
It was suggested by @porteron. It's our own code so it doesn't matter what exactly we choose as long as it is used on both client and server side.